### DISCUSSION TITLE
Ability to add same show multiple times, with languages variations

### DISCUSSION THREAD

**Discussion by mysterfr** [pymedusa/Medusa#8731] [Ideas & Feature requests]
**Situation:**
I'm French, and I often download series in multiple languages.

For example, let's consider I'm downloading series "Random Serie Name":
- first in "VOSTFR" language (equivalent to SUBFRENCH), means original dubbing (usually english) with French subtitles.
- then quite a while after that (usually about 6 months-1year later due to dubbing constraints), I download the very same show, this time dubbed in FRENCH. Usually because my girlfriend watches the same shows as I do, but wants to watch them in French. And yes, I'm usually one season ahead of her on our common shows...
- On top of that, for certain shows again, I even re-download the same episodes in "MULTi" (so a single episode contains both English and French dubbings, plus matching subtitles). I then erase the "VOSTFR" and "FRENCH" versions to only keep the "MULTi" one.

Each version of the show's episodes is stored in a different tree:
- /media/Series/VOSTFR/Random.Serie.Name/Season.01
- /media/Series/FRENCH/Random.Serie.Name/Season.01
- /media/Series/MULTi/Random.Serie.Name/Season.01

**Solution proposal**
I would like to be able to add a given show multiple time in Medusa, with languages variants.
With the above example, I would add:
- "Random Series Name **- VOSTFR**"
- "Random Series Name **- FRENCH**"
- "Random Series Name **- MULTi**"

Each variant of the serie has keywords I can prefer and forbid.
For instance:
- for the "VOSTFR" version, I'd set "VOSTFR" and "SUBFRENCH" as prefered keywords, and forbid "FRENCH" and "MULTi"
- for the "MULTi" version, I'd set "MULTi" as prefered keyword, and forbid "VOSTFR". Here I would not forbid the "FRENCH" keyword as sometimes releases are tagged with "MULTi.TRUEFRENCH".
- for the "FRENCH" version, I'd set "FRENCH" as prefered keyword, and forbid "MULTi" and "VOSTFR"

And each variant has it's own lookup folder as described above.

I could not imagine any other way to handle that.
I'm already taking care of this with Flexget : I'm using multiple configuration files (one per target language) and while it is a bit tidious to maintain, it works quite well.

As a bonus, I would like to be able to replicate the same thing, but with qualities.
For instance, be able to download "Random Series Name" in French, both in SD **and** in HD.

In the end, the Medusa list show could end up looking like this:
- "Random Series Name - VOSTFR" - HD
- "Random Series Name - FRENCH" **- HD**
- "Random Series Name - FRENCH" **- SD**
- "Random Series Name - MULTi" - HD

**Comment by ragepaw-git** [comment]
I would like to see this but for slightly different reasons. I would like to keep multiple resolutions of shows. I currently have HD and 4K versions of some shows, and I'm doing it manually. I would prefer to do this without a second instance,

**Reply by ragepaw-git** [reply]
Me too for the same reason. I'm currently running 2 instances in Docker. One for HD and one for 4K.

It would be awesome if I could drop to one instance.

**Reply by ragepaw-git** [reply]
Wow. I replied to my own post and didn't notice.
